### ISSUE TITLE
Overindentation when using "class" as a key in an object

### ISSUE THREAD

**Issue by laggingreflex** [beautify-web/js-beautify#1838]
Expected: 

```js
{
    class: {
        a: 1,
        b: 2,
        c: 3,
    }
}
```

Actual:

```js
{
    class: {
        a: 1,
            b: 2,
            c: 3,
    }
}
```

**Comment by bitwiseman** [comment]
Another keyword needing a special case.

**Comment by mhnaeem** [comment]
Fix for this issue released with #2013
